data-prepper-api/src/main/java/org/opensearch/dataprepper/metrics/MetricNames.java

```diff
@@ -86,4 +86,6 @@ private MetricNames() {}
      * Delimiter used to separate path components in metric names.
      */
     public static final String DELIMITER = ".";
+
+    public static final String LATENCY = "latency";
```

Comment:
Should we use a different name here? The term latency is ambiguous as there are numerous places where latency exists.

It appears that this is the time before events are read from the buffer. Some name ideas:

* `bufferReadLatency`
* `readLatency`
* `timeEventIdle`
* `atRestTime` (to put it in opposition to in-flight)